Octokit.Tests.Integration/Clients/RepositoriesClientTests.cs

```diff
@@ -1697,57 +1697,42 @@ public class TheTransferByNameMethod
                 var transferred = await github.Repository.Get(newOwner, context.RepositoryName);
             }
         }
-
+        
         [IntegrationTest]
-        public async Task CanTransferUserRepoToOrg()
+        public async Task TransfersFromOrgToUserWithId()
         {
             var github = Helper.GetAuthenticatedClient();
             var newRepo = new NewRepository(Helper.MakeNameWithTimestamp("transferred-repo"));
-            var newOwner = Helper.Organization;
-            using (var context = await github.CreateRepositoryContext(newRepo))
+            var newOwner = Helper.UserName;
+            using (var context = await github.CreateRepositoryContext(Helper.Organization, newRepo))
             {
                 var transfer = new RepositoryTransfer(newOwner);
-                await github.Repository.Transfer(context.RepositoryOwner, context.RepositoryName, transfer);
-                var transferred = await github.Repository.Get(newOwner, context.RepositoryName);
+                await github.Repository.Transfer(context.RepositoryId, transfer);
+                var transferred = await github.Repository.Get(context.RepositoryId);
```

Comment:
eventhough this would throw an exception if the repo didnt exist under the requested owner (thus failing the test), I think i'd still prefer to see some kind of `Assert` in each test, for consistency.

eg: `Assert.Equal(newOwner, transferred.Owner.Login);`